plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -2847,6 +2847,50 @@ public void testBasicTableStatistics()
         dropTable(tableName);
     }
 
+    /**
+     * @see TestIcebergAnalyze
+     */
+    @Test
+    public void testBasicAnalyze()
+    {
+        String tableName = "test_analyze";
+        assertUpdate("CREATE TABLE " + tableName + " AS SELECT * FROM tpch.tiny.region", 5);
+
+                        // no NDV information
```

Comment:
nit: The indentation seems wrong. Please ignore if it's intentional.

Reply:
thanks, fixed